File: JavaScript/002-map.js
// Given an array of integers, return a new array with each value doubled.
// For example:
// [1, 2, 3] --> [2, 4, 6]


function maps(arrayOfNumbers){
    const newArrayDoubled = [];
    
    for(let i = 0; i < arrayOfNumbers.length; i++) {
        const number = arrayOfNumbers[i] * 2;
        newArrayDoubled.push(number)
    }

    return newArrayDoubled;
}


console.log(maps([1, 2, 3])) // [2, 4, 6]

File: Python/002-map.py
# Given an array of integers, return a new array with each value doubled.
# For example:
# [1, 2, 3] --> [2, 4, 6]


def maps(a):
    return [number * 2 for number in a]


if __name__ == '__main__':
    print(maps([1, 2, 3])) # [2, 4, 6]
